packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -153,6 +154,19 @@ const App = (props) => {
         location?.pathname
     )
 
+    // TODO: Remove test logs before merging
+    const logger = createLogger('retail-react-app')
```

Comment:
How about we take this a step further? We can have one logger instance for the entire package. So you don't need to call `createLogger` in every file that you want to use it.

Something like this:
```js
// In app/logger.js
import createLogger from '../../pwa-kit-runtime/src/utils/logger'

const logger = createLogger('retail-react-app')
export default logger

// In the App component
import logger from '@salesforce/retail-react-app/app/logger'
logger.log('hello world')
```

Reply:
I like the idea of creating a unique logger per package. I made the change.